src/webhooks/registry.ts

```diff
@@ -191,9 +199,11 @@ const WebhooksRegistry: RegistryInterface = {
       const {node} = checkResult.body.data.webhookSubscriptions.edges[0];
       let endpointAddress = '';
       if ('endpoint' in node) {
-        endpointAddress = node.endpoint.__typename === 'WebhookHttpEndpoint'
-          ? node.endpoint.callbackUrl
-          : node.endpoint.arn;
+        if (node.endpoint.__typename === 'WebhookHttpEndpoint') {
+          endpointAddress = node.endpoint.callbackUrl;
+        } else if (node.endpoint.__typename === 'WebhookEventBridgeEndpoint') {
+          endpointAddress = node.endpoint.arn;
+        }
```

Comment:
Potentially silly question: is this just changing this from a ternary, or are we avoiding returning the incorrect thing for when we have a pub/sub hook with the more explicit `else if`?

Reply:
Maybe another `switch` here would make that clearer?